doc/softphone.md

```diff
@@ -70,7 +70,7 @@ Alice                     Push Notifications    WebRTC SDK
 
 \^ denotes optional API usage outside of the SDK to get complete conversation details
 
-## Usage
+## Example Usage
```

Comment:
It is a pretty small file still, but it may be worth it to throw a table of content at the top for these `##` sub headers.